src/indexes.ts

```diff
@@ -69,32 +69,34 @@ class Indexes extends MeiliAxiosWrapper {
         params.limit = options.limit
       }
       if (options.attributesToRetrieve) {
-        params.attributesToRetrieve = options.attributesToRetrieve.join()
-      }
-      if (options.attributesToSearchIn) {
-        params.attributesToSearchIn = options.attributesToSearchIn.join()
+        if (Array.isArray(options.attributesToRetrieve)) {
+          params.attributesToRetrieve = options.attributesToRetrieve.join()
```

Comment:
Be careful, the join method without parameters makes joins with `, ` and not `,`. we didn't accept spaces in query parameters.

Reply:
It might have changed: https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Objets_globaux/Array/join

In either case lets not take the risk! I changed it everywhere 👍